packages/create-react-native-library/templates/common/example/metro.config.js

```diff
@@ -14,9 +14,9 @@ module.exports = {
   watchFolders: [root],
 
   // We need to make sure that only one version is loaded for peerDependencies
-  // So we blacklist them at the root, and alias them to the versions in example's node_modules
+  // So we block them at the root, and alias them to the versions in example's node_modules
   resolver: {
-    blacklistRE: blacklist(
+    exclusionListRE: exclusionList(
```

Comment:
I just tested this and this needs to be 
```
blacklistRE: exclusionList(
```

otherwise it doesn't work

Reply:
Are you sure it's `blacklist` and not a `blockList`?
Please check 
[this line](https://github.com/facebook/metro/blob/1b905b60e397f2213f4ad20eb8201516b55affab/packages/metro-config/src/defaults/index.js#L39) in metro.

I can see that this is quite messy, as the history of the *packages/metro-config/src/defaults/index.js* contains two changes, and the issue which this pr closes points to the first one, which leaves the `blacklistRE` where it was, but another change 
[Add support for blockList and array of regexs ](https://github.com/facebook/metro/commit/94c0b541b4bfa17aee4efa0f1969565522ce830d#diff-1a3c1a959bb8c4e2e9743c03cb7a6d0c56648ffcfe129a11b9090bfc139622dd) makes it a `blockList` instead of `blacklist`.
Are you sure you have tested it on the latest of metro?

Reply:
Yes im sure. Checkout my repo - https://github.com/walterholohan/react-native-crisp-chat-sdk/blob/master/example/metro.config.js.

Ive tested the above and it works

Reply:
Sure! Then I will deliver the fix within a minute.
Thank you very much for your interest, @walterholohan!